string instruction
geometry_msgs/Pose[] goal_path_poses_list
bool blocking
---
bool success
